app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,8 @@
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
 
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload', defer: true %>
+    <script src="https://rawgit.com/enyo/dropzone/master/dist/dropzone.js"></script>
```

Comment:
@zyehia -- I assume we'll need to figure out a way to get this functionality into the app in a less hacky way? But for now, this works.

Reply:
yep, agree